src/JuliusSweetland.OptiKey.Core/OptiKeyApp.cs

```diff
@@ -785,7 +785,7 @@ protected static ICalibrationService CreateCalibrationService()
                         Settings.Default.PointSelectionTriggerFixationLockOnTime,
                         Settings.Default.PointSelectionTriggerFixationCompleteTime,
                         Settings.Default.PointSelectionTriggerLockOnRadiusInPixels,
-                        Settings.Default.PointSelectionTriggerFixationRadiusInPixels,
+                        Settings.Default.PointSelectionTriggerLockOnRadiusInPixels,
```

Comment:
This doesn't look right - you're passing the lock on radius (px) twice. The signature being called is:

public PointFixationSource(
            TimeSpan lockOnTime,
            TimeSpan timeToCompleteTrigger,
            double lockOnRadius,
            double fixationRadius,
            IPointSource pointSource)

Reply:
@JuliusSweetland I will implement this differently in my final version, but for now, just know that it is on purpose to get the correct experience.